### PR TITLE
chore(docs): getting-started-docs

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

File: docs/design/palette/guidelines-usage-and-mapping.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+Colors have assigned meanings and are used consistently throughout Vivid to set expectations of meaning for users. Color is used sparingly and intentionally to reinforce hierarchies and to create clear modes of communication.
+
+- Black - Accent
+- Purple - CTA
+- Magenta - Announcement
+- Blue - Information
+- Red - Alert
+- Green - Success
+- Yellow - Warning
+
```
Comment (rinaok) [inline docs/design/palette/guidelines-usage-and-mapping.md:54]: @AyalaBu We are missing this picture here:
Screen Shot 2022-09-07 at 19 12 56

**Comment by rachelbt** [comment]
I didn't go over all yet but I don't think that "Welcome to our new and improved VIVID UI Kit!" and its content is if for the Vivid-next homepage. 
I think the site is mainly for developers and this is not relevant for them. vivid@next is not a UI kit for developers. Its an "Essential UI web components" as appearing in the homepage now.

Also not sure that this is relevant for this PR but after pressing a menu item that is a "sub item" the item is closed and no way to see other elements.
before pressing Design Tockens:
image

after:
image
can't open or close it

**Comment by rinaok** [comment]
> I didn't go over all yet but I don't think that "Welcome to our new and improved VIVID UI Kit!" and its content is if for the Vivid-next homepage. I think the site is mainly for developers and this is not relevant for them. vivid@next is not a UI kit for developers. Its an "Essential UI web components" as appearing in the homepage now.
> 
> Also not sure that this is relevant for this PR but after pressing a menu item that is a "sub item" the item is closed and no way to see other elements. before pressing Design Tockens: image
> 
> after: image can't open or close it

@rachelbt Can you suggest inside the text what changes do you think we should make? It will be more clear.
About the opening bug, we know about it. I don't know how to fix it yet.

**Comment by AyalaBu** [comment]
@rinaok I completely understand your dilemma, 
but it's just an explainer, so maybe there is no need to be bothered.
In the meantime maybe we should leave it as it is, 
And if a when an opportunity presents itself write another content.
Besides, maybe in order to create a content that can suit the developers we should know what do you this is the content that can actually be useful. What do you think? @rinaok @yinonov @rachelbt

**Comment by rachelbt** [comment]
@rinaok  @AyalaBu 
Summarising our decisions from the talk:
Our main audience is developers but it is important to have data for designers in the Vivid-3 as well.

- We will add a category for designer where we will add all the readme for the designers.
- The Home -Page of the docs will be more "general" and not about installation - with links to point designers & developer to their getting started. @AyalaBu - this is on you :)

references to other style-guide:
https://nordhealth.design/start/
https://carbondesignsystem.com/developing/get-started/

**Comment by AyalaBu** [comment]
@rachelbt @rinaok @yinonov created a branch of the original UI Kit 2.0 file 
in order to be able to work on the documentation there

**Comment by rinaok** [comment]
> @rachelbt @rinaok @yinonov I created a branch of the original UI Kit 2.0 file in order to be able to work on the documentation there

@AyalaBu Can you please add the link here?

**Comment by AyalaBu** [comment]
@rinaok @rachelbt @yinonov 
Voila!

https://www.figma.com/file/r8OYLPmRmDqfLKt6KHdJDB/Vivid-UI-Kit---2.0?node-id=76%3A1267

**Comment by yinonov** [comment]
please reopen if still relevant

**Pull request closed without merging**